skyportal/tests/frontend/test_scanning_page.py

```diff
@@ -840,7 +840,7 @@ def test_user_without_save_access_cannot_save(
         f'//button[@name="initialSaveCandidateButton{public_candidate.id}"]'
     )
     driver.scroll_to_element_and_click(save_button)
-    driver.wait_for_xpath("//*[contains(.,'Insufficient permissions')]")
+    # driver.wait_for_xpath("//*[contains(.,'Insufficient permissions')]")
```

Comment:
Should this just be deleted before merging if its not used anymore ?

Reply:
yeah